ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -558,10 +558,17 @@
 magenta_squad = pytest.mark.magenta_squad
 orange_squad = pytest.mark.orange_squad
 purple_squad = pytest.mark.purple_squad
-red_squad = pytest.mark.red_squad
 turquoise_squad = pytest.mark.turquoise_squad
 yellow_squad = pytest.mark.yellow_squad
 
+# Most of the noobaa related tests are under red_squad,
+# so this composition allows us to capture those while
+# skipping the RGW-only tests when the noobaa marker is specified
+red_squad = compose(pytest.mark.red_squad, pytest.mark.noobaa)
+
+# To decorate noobaa-related tests that may not be under red_squad
+noobaa = pytest.mark.noobaa
```

Comment:
Rebase and use one from:
https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/framework/pytest_customization/marks.py#L66

Reply:
Rebased and then changed to `mcg = pytest.mark.mcg` as we discussed in the automation meeting